### PR TITLE
added missed switcher for RBAC-disabled deployment

### PR DESCRIPTION
**Pull request by glebsa8**
#### What this PR does / why we need it:
This PR adds missed switcher for RBAC-disabled deployment of chart `stable/jaeger-operator`. 

Without fix:

```
» helm install stable/jaeger-operator --name jaeger-operator --set rbac.create=false --set serviceAccount.create=false
Error: release jaeger-operator failed: roles.rbac.authorization.k8s.io "jaeger-operator" is forbidden: attempt to grant extra privileges: [{[*] [io.jaegertracing] [*] [] []} {[*] [] [pods] [] []} {[*] [] [services] [] []} {[*] [] [endpoints] [] []} {[*] [] [persistentvol
umeclaims] [] []} {[*] [] [events] [] []} {[*] [] [configmaps] [] []} {[*] [] [secrets] [] []} {[*] [apps] [deployments] [] []} {[*] [apps] [daemonsets] [] []} {[*] [apps] [replicasets] [] []} {[*] [apps] [statefulsets] [] []} {[*] [extensions] [ingresses] [] []}] user=&
{system:serviceaccount:kube-system:tiller 71c9b746-b755-11e8-bdd9-3a88d844dbf2 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[clusterroles.rbac.authorization.k8s.io "cluster-admin" not found]
```

With fix:

```
» helm install Repos/helm-charts/stable/jaeger-operator/ --name jaeger-operator --set rbac.create=false --set serviceAccount.create=false --wait
NAME:   jaeger-operator
LAST DEPLOYED: Thu Nov 22 17:57:09 2018
NAMESPACE: default
STATUS: DEPLOYED

RESOURCES:
==> v1/Deployment
NAME             AGE
jaeger-operator  5s

==> v1/Pod(related)

NAME                              READY  STATUS   RESTARTS  AGE
jaeger-operator-766c69b5c7-dg4qf  1/1    Running  0         5s


NOTES:
jaeger-operator is installed.


Check the jaeger-operator logs
  export POD=$(kubectl get pods-l app.kubernetes.io/instance=jaeger-operator -lapp.kubernetes.io/name=jaeger-operator --namespace default --output name)
  kubectl logs $POD --namespace=default
```

#### Special notes for your reviewer:
@cpanato, please review!

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by glebsam** [comment]
ping @cpanato

File: stable/jaeger-operator/templates/role.yaml [hunk 1]
```diff
@@ -41,3 +42,4 @@ rules:
   - ingresses
   verbs:
   - "*"
+{{- end }}
```
Comment (cpanato) [inline stable/jaeger-operator/templates/role.yaml]: add a blank line here 😄

Comment (glebsa8) [reply stable/jaeger-operator/templates/role.yaml]: Done!

**Pull request merged**